conf/solr/conf/managed-schema.xml

```diff
@@ -200,6 +200,42 @@
     <field name="ratings_count_4" type="pint"/>
     <field name="ratings_count_5" type="pint"/>
 
+    <!-- Trending related values-->
+    <field name = "trending_score_hourly_0" type="pint" indexed="false" stored ="false"/>
```

Comment:
Note: The field settings here are something we might want to investigate if we see perf issues.